app/pages/embed/[chartId].tsx

```diff
@@ -26,7 +27,18 @@ export const getServerSideProps: GetServerSideProps<PageProps> = async ({
 
   if (config && config.data) {
     // TODO validate configuration
-    return { props: { status: "found", config } };
+    return {
+      props: {
+        status: "found",
+        config: {
+          ...config,
+          data: {
+            ...config.data,
+            chartConfig: migrateChartConfig(config.data.chartConfig),
+          },
+        },
+      },
+    };
```

Comment:
Could we have the migration directly in getConfig ? This way we would not have to repeat it everywhere we use getConfig ?